src/components/TextareaField/TextareaField.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { generateSnapshots } from '@chanzuckerberg/story-utils';
+
+import { render, screen } from '@testing-library/react';
```

Comment:
Reminder that we have access to both `@testing-library/react` and the similar export from a storybook sub-package. When writing tests here, we don't want the storybook exports, b/c they are wrapped with additional functionality that may throw warnings.